blog-posts/case-fc-bayern.md

```diff
@@ -0,0 +1,66 @@
+---
+path: "/blog/case-fc-bayern/"
+date: "2022-02-21"
+title: "Relaunch FC Bayern Website"
+featuredImage: images/fcb/hero.png
+author: Fabian Dietenberger
+authorSummary: "Frontend Engineer, Satellytes"
+seoMetaText: Our Approach to relaunch the Frontend of the Fc Bayern Website.
+teaserText: Our Approach to relaunch the Frontend of the Fc Bayern Website.
+---
+<sectionheader>
+ Our task was the frontend relaunch of the FC Bayern website. We took a holistic approach and created a design system with a solid set of components and guidelines to unify the look and feel of forthcoming digital experiences and introduced a headless architecture as a modern foundation on top of the existing CMS to be prepared for everything to come.
+</sectionheader>
+
+#### [www.fcbayern.com](www.fcbayern.com)
+
+<br><br>
+
+<stat title="Team Size" content="6 people"></stat>
+<stat title="Start" content="December 2018"></stat>
+<stat title="End" content="Ongoing"></stat>
+
+
+![](images/fcb/overview.png)
+
+### Approach
+
+We started with an inventory of all applications and platforms in the FC Bayern München ecosystem and a thorough benchmarking of other sports clubs with the goal to create the best soccer club page out there and also a solid fundament to build and expand on after the initial launch with many more exciting features to come.
+We analyzed all features and funnels on the then current FC Bayern München website, extended the style guide with new patterns and guidelines that can serve all use cases and created a design system as a single source of truth – also for other agencies and future products.
+We created a new architecture for the website that can serve all common and special use cases of an international soccer club like FC Bayern, e.g. live-features, special announcements, a comprehensive match center with detailed stats that satisfy even the biggest fans and formats that enable FC Bayern to interact with their diverse audience.
+We created a powerful theming system with light/dark support that makes it easy to adapt the implementation to changing designs required by the different teams and companies within the FC Bayern universe.
```

Comment:
You need to introduce extra line breaks to create paragraphs. Right now it's a blog of text and looks not good nor expected.

![Screenshot 2022-02-22 at 06 02 52](https://user-images.githubusercontent.com/1701755/155066344-657c137c-97c6-4dac-8256-931b78e9f98c.png)


```suggestion
We started with an inventory of all applications and platforms in the FC Bayern München ecosystem and a thorough benchmarking of other sports clubs with the goal to create the best soccer club page out there and also a solid fundament to build and expand on after the initial launch with many more exciting features to come.

We analyzed all features and funnels on the then current FC Bayern München website, extended the style guide with new patterns and guidelines that can serve all use cases and created a design system as a single source of truth – also for other agencies and future products.

We created a new architecture for the website that can serve all common and special use cases of an international soccer club like FC Bayern, e.g. live-features, special announcements, a comprehensive match center with detailed stats that satisfy even the biggest fans and formats that enable FC Bayern to interact with their diverse audience.

We created a powerful theming system with light/dark support that makes it easy to adapt the implementation to changing designs required by the different teams and companies within the FC Bayern universe.
```